### ISSUE TITLE
Quotes in sketch filenames are not escaped correctly

### ISSUE THREAD

**Issue by alokjethanandani** [avocode/avocode#1592]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.8`
Avocode UID: 166644
```
Error: Command failed: "/Applications/Sketch.app/Contents/Resources/sketchtool/bin/sketchtool" dump --overwriting --use-id-for-name --formats="png" "/Users/alokjethanandani/Dropbox (Fundbox)/Projects (1)/096 Fundbox Pay/Q317 Initiatives/Finalized Designs/LAB-645 "To" Field Autocomplete.sketch" > "/var/folders/tn/695wxzsj6zz4vbs0fv2_zm2w0000gn/T/sketch-avocode-full-dump.1503610926511.json"
2017-08-24 14:42:06.562 sketchtool[57931:17860667] -[ECCommandLineEngine outputError:format:]: unrecognized selector sent to instance 0x7f83d2e12090
Command threw exception

(ECCommandLineDomain:5)


    at ChildProcess.exithandler (child_process.js:223:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:885:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
```

**Comment by jakubzitny** [comment]
Hi @alokjethanandani, thank you for reporting this error. I see where the problem here is, it's our fault and it will be fixed for next release. We are incorrectly escaping the quotes in filenames.

As a workaround for now, please rename your file to not conain any quotes and sync again, please.

**Comment by jakubzitny** [comment]
Hi there, this is fixed in latest Avocode `2.15.0`. You can download it from [avocode.com/download](https://avocode.com/download) or use auto-update to install it. Please give it a try and let us know if you have any more issues.
